pkg/upstream/replicated.go

```diff
@@ -315,30 +313,30 @@ func parseReplicatedURL(u *url.URL) (*ReplicatedUpstream, error) {
 	return &replicatedUpstream, nil
 }
 
-func getSuccessfulHeadResponse(replicatedUpstream *ReplicatedUpstream, license *kotsv1beta1.License) (*kotsv1beta1.License, error) {
```

Comment:
This change doesn't do anything, but I kept it because this way it's not misleading.